docs/TOKENS.md

```diff
@@ -13,133 +13,141 @@ Spoilers. My love for Astro has grown deeper.
 
 ## Prerequisites
 1. NodeJS and Astro installed.
-2. A developer account with [Agora](https://console.agora.io/).
+2. Know how to send a POST request using Postman
+3. A developer account with [Agora](https://console.agora.io/).
 
 ## Project Setup
 To create a new Astro project, run `npm create astro@latest`. Follow all the default options during the setup process, except use an empty template. This way, our project will be free of distractions. Then, we must install the Agora Token package to generate our tokens. You can do that using `npm i agora-token`.
 
-To generate a token, we need to use our Agora App ID and App Certificate. The App Certificate is our "secret" key and should be hidden from public view. We will use a `.env` file to store our sensitive data. When we generated our Astro project, the `.env` was already listed in the `.gitignore`, which means it will not be pushed to GitHub. In the `.env`, add the project `APP_ID` and `APP_CERTIFICATE`, which you can find in your Agora Console.
+To generate a token, we need to use our Agora App ID and App Certificate. The App Certificate is our "secret" key and should be hidden from public view. We will use a `.env` file to store this sensitive data. When we generated our Astro project, the `.env` was already listed in the `.gitignore`, which means it will not be pushed to GitHub. In the `.env`, add the project `APP_ID` and `APP_CERTIFICATE`, which you can find in your Agora Console.
```

Comment:
The mixed tenses in this sentence make it hard to read. Jumping from future tense to past tense
>  We will use a `.env` file to store
> When we generated our Astro project, the `.env` was already listed in the `.gitignore`,
Consider adding a transition or modifiying the tense in the first statement.

Reply:
Updated to: 

It is a commmon practice to use an `.env` file to store this sensitive data. When we generate our Astro project, the `.env` is already listed in the `.gitignore`, which means it will not be pushed to GitHub.